lib/workers/repository/init/merge.ts

```diff
@@ -278,3 +281,37 @@ export async function mergeRenovateConfig(
   }
   return returnConfig;
 }
+
+// needed when using portal secrets for npmToken
+export function setNpmTokenInNpmrc(config: RenovateConfig): void {
+  if (!is.string(config.npmToken)) {
+    return;
+  }
+
+  const token = config.npmToken;
+
+  if (!is.string(config.npmrc)) {
+    logger.debug('Adding npmrc to config');
+    config.npmrc = `//registry.npmjs.org/:_authToken=${token}\n`;
+    delete config.npmToken;
+    return;
+  }
+
+  /* eslint-disable no-template-curly-in-string */
+  if (config.npmrc.includes('${NPM_TOKEN}')) {
```

Comment:
```suggestion
  if (config.npmrc.includes(`\${NPM_TOKEN}`)) {
```
should solve the eslint issue